### PR TITLE
add action

### PR DESCRIPTION
**Pull request by morri-son**
## Description

add action that moves items from done to closed until github API is able to manage project v2 (beta)

### CODE REVIEW

File: .github/.reuse/dep5
Comment (hilmarf) [inline .github/.reuse/dep5:1]: Is the file accidentally removed? Or by intention?

File: .github/workflows/update-project-item.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  for (const itemNumber of itemsToUpdate) {
+    const itemSelector = `[data-item-id="${itemNumber}"]`;
+    const item = await page.waitForSelector(itemSelector);
+    const statusDropdown = await item.$('.js-project-status-field');
+    await statusDropdown.click();
+    const statusOption = await page.waitForSelector(`.js-project-status-field option[value="${newStatus}"]`);
+    await statusOption.click();
+  }
+
+  await browser.close();
```
Comment (hilmarf) [inline .github/workflows/update-project-item.js:31]: argh... what a quirks workaround

**Review by hilmarf — APPROVED** [review]

File: .github/workflows/update-project-item.js (hunk 1 above)
Comment (morri-son) [reply .github/workflows/update-project-item.js:31]: I guess this was copy and paste error before. .reuse needs to reside in the root folder, not under .github

**Comment by Skarlso** [comment]
You can use GitHub graphql from GitHub actions to manage projects. I've done that before.

**Comment by morri-son** [comment]
> You don't need this JavaScript monstrosity. 🤣

Not really. This a a project V2 (beta). And the PO of the new project board decided to release it without exposing the GraphQL API for mutating custom fields. POs move issues on the board by hand 🤣. I would not not use a headless browser to simulate that if I could do a simple gh project...

**Comment by Skarlso** [comment]
I'm not following sorry. 

So which action can't you do with the ProjectV2 graphQL API currently? https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects

**Comment by morri-son** [comment]
> I'm not following sorry.
> 
> So which action can't you do with the ProjectV2 graphQL API currently? https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects

@Skarlso , in principle I want to move Items from column "Done" to "Closed" if the date for the last update of an item in the done column is older than two weeks. If your https://github.com/Skarlso/caretaker/blob/main/pkg/client/client.go can be used for that, that would be cool :-)

**Comment by Skarlso** [comment]
Yep, that should be possible. I'll take a look! :)

**Comment by morri-son** [comment]
[like]  Morrison, Gerald reacted to your message:
________________________________

File: .github/workflows/update-project-item.js (hunk 1 above)
Comment (stb1337) [reply .github/workflows/update-project-item.js:31]: hacky... I like it ;)

**Comment by Skarlso** [comment]
@morri-son @hilmarf updated to use caretaker. :)

_Note_: I don't think GITHUB_TOKEN will work as it doesn't have projectv2 permissions yet. 🤔

**Comment by Skarlso** [comment]
@morri-son I'm gonna merge this and then play around with running the action.

**Pull request merged**